packages/cheatsheet/jest.config.js

```diff
@@ -2,4 +2,5 @@
 module.exports = {
   preset: "ts-jest",
   testEnvironment: "jsdom",
+  testPathIgnorePatterns: ["/node_modules/", "/out/"],
```

Comment:
not sure why I had to add this one now, but seems to fix things and I confirmed that the tests are still running in this dir

Reply:
I think this is a sign we should be turning on noEmit here.